### PR TITLE
Tools: use more common parameters from "base" vehicle files

### PR DESCRIPTION
**Pull request by peterbarker**

### CODE REVIEW

File: Tools/autotest/default_params/copter-heli-dual.parm [hunk 1]
```diff
@@ -1,83 +1,2 @@
...
-RC8_TRIM        1500.000000
-WP_YAW_BEHAVIOR 2
-IM_ACRO_COL_EXP 0.2
-IM_STAB_COL_1   400
-IM_STAB_COL_2   600
-IM_STAB_COL_3   700
-IM_STAB_COL_4   900
-
-
+ATC_PIRO_COMP   0   # should this line be removed?
```
Comment (peterbarker) [inline Tools/autotest/default_params/copter-heli-dual.parm]: I've already put the question to @R-Lefebvre

**Comment by peterbarker** [comment]
@OXINARF I've done as you suggested and just made coax inherit from single.

It was worse than not reverting, actually; unsurprisingly there were heli parameters not understood by the base binary....

If we ever stop having a separate heli binary we should revisit for sure.

**Comment by khancyr** [comment]
Looks good to me, for rover part !

File: Tools/autotest/default_params/copter-heli-dual.parm (hunk 1 above)
Comment (OXINARF) [inline Tools/autotest/default_params/copter-heli-dual.parm]: Any answer from @R-Lefebvre?

**Comment by OXINARF** [comment]
I've talked to @R-Lefebvre and he said he isn't sure about the answer. He says that the ATC_PIRO_COMP option is really only useful for sport flying and so he isn't sure on turning it on/off on both Heli and Heli Dual. Given this, I think the best is to leave the parameter as it was in both Heli and Heli Dual. I've left the comment in the Heli Dual parameter file since there is no factual answer to it.

Rebased (with correcting the Rover Skid parameter file) and merged, thanks!

**Pull request closed without merging**